tests/ui/needless_range_loop.stderr

```diff
@@ -61,6 +61,7 @@ error: the loop variable `i` is only used to index `vec2`
 LL |     for i in 0..vec.len() {
    |              ^^^^^^^^^^^^
    |
+   = note: suggestion might skip expected iterations if `vec2.iter().count()` is less than `vec.len()`
```

Comment:
If the suggestion skips some iteration, then that means it's a change in behavior. Wouldn't it be a bad suggestion issue and should be labeled as such?